### ISSUE TITLE
[D7] Theme sign-in form.

### ISSUE THREAD

**Issue by fureigh** [18F/web-design-standards-drupal-d7#42]
Stub.

**Notes:**
- Check whether #35 addresses this sufficiently.
- See #15 for complete list of components.
- See #43 for corresponding D8 issue.

**Comment by brockfanning** [comment]
I don't think this required any additional theming, beyond what USWDS does out of the box, but in #66 I did remove some unnecessary help text, and added the ability to log in using email or username (per the [guidelines](https://standards.usa.gov/components/form-templates/#sign-in-form).
